frontend/src/metabase/querying/notebook/components/NotebookDataPicker/NotebookDataPicker.tsx

```diff
@@ -44,10 +45,15 @@ export function NotebookDataPicker({
   table,
   databaseId,
   placeholder = title,
-  hasMetrics,
+  hasMetrics = false,
```

Comment:
changes the type within the component from `boolean | undefined` to `boolean` just to make things a little easier